docker-compose.yml

```diff
@@ -155,7 +155,7 @@ services:
       - 80:80
       - 443:443
     volumes:
-      - ./config/traefik:/etc/traefik
+      - ${DATA_DIR:-./data}/config/traefik:/etc/traefik
```

Comment:
Sure about this one? This is the predefined config and I don't think we want people to edit this.

Reply:
I did not change the files that are mounted, only their location. It was possible in the past to edit the `traefik` configuration by editing `./config/traefik`.

I do want to at least move this config file inside `./data` so that a `git add .` won't add wrong files to a commit. @ulope any other opinions?

Reply:
@palango sorry, I thought these files were generated, you're correct they should not be in the `data` folder.